cirq/work/sampler.py

```diff
@@ -13,17 +13,59 @@
 # limitations under the License.
 """Abstract base class for things sampling quantum circuits."""
 
-from typing import List, Optional, TYPE_CHECKING, Union
+from typing import Dict, List, Optional, TYPE_CHECKING, Union
 import abc
 
+import numpy as np
 import pandas as pd
 
-from cirq import study
+from cirq import ops, study
+from cirq.work import group_settings_greedy, observables_to_settings
+from cirq.work.observable_measurement import (
+    _get_params_for_setting,
+    _with_parameterized_layers,
```

Comment:
This is bad practice to import private functions from other modules.  It breaks naming assumptions.  We should either change the name to indicate that it is no longer private or remove the import (or create a new method that is public in those modules).

Reply:
I agree, although this behavior is consistent with the observable measurements imports (all files are under `cirq/work`):
https://github.com/quantumlib/Cirq/blob/845836ac5fa3cf2bba0258c2e01169a28eb1a2fd/cirq/work/observable_measurement_data.py#L23-L29

Should I send a "prequel" PR that fixes this before moving forward with this one?